papergres.go

```diff
@@ -407,20 +407,31 @@ func (db *Database) GenerateInsert(obj interface{}) *Query {
 	return db.Schema("public").GenerateInsert(obj)
 }
 
-// Insert inserts the passed in object
+// Insert inserts the passed in object in DB.
+//
+// NOTE: It does not account for client side generated value for a
+// primary key and expects that the logic for populating value
+// of primary key should reside in the database as sequence.
+//
+// DO NOT use Insert() if you wish to populate a client side generated
+// value in primary key, use InsertWithPK() instead.
 func (s *Schema) Insert(obj interface{}) *Result {
-	sql := insertSQL(obj, s.Name)
-	args := insertArgs(obj)
+	sql := insertSQL(obj, s.Name, false)
```

Comment:
Based on this Papergres provides no way to change the DB table name to whatever you'd like. Is it that the name of the Struct has to be what determines the name of the table being inserted to?

Reply:
Good catch! And that was one of the design decisions while creating papergres, so that structs can have a direct co-relation with the table they are referring to. Everything else then just seems synonymous.
And the ideology is the other way around, the name of a table should decide the name of the struct to have that 1:1 co-relation.

Reply:
In other words, this leaves a situation where if I would like to have a shadow table that records the history of a table then I would need two different Structs. Or I would be forced to use PGAudit?

Please note: I'm not saying we should change this and that PGAudit and shadow tables have drastically different use cases.